chat.tox.qTox.json

```diff
@@ -0,0 +1,44 @@
+{
+	"app-id": "chat.tox.qTox",
+	"runtime": "org.kde.Platform",
+	"sdk": "org.kde.Sdk",
+	"runtime-version": "5.10",
+	"command": "qtox",
+	"finish-args": [
+		"--share=network",
+		"--socket=pulseaudio",
+		"--socket=wayland",
+		"--socket=x11",
+		"--filesystem=host",
+		"--device=all"
+	],
+	"build-options": {
+		"cflags": "-O3 -DSQLITE_HAS_CODEC",
```

Comment:
I believe `-DCMAKE_BUILD_TYPE=Release` in `config-opts` is the better way to set optimization level?